api/app/signals/apps/email_integrations/templates/email/forward_to_external_reaction_received.html

```diff
@@ -0,0 +1,52 @@
+Geachte behandelaar,
```

Comment:
Please remove the plain text from the HTML

Reply:
done